src/utils/propsBinder.js

```diff
@@ -14,34 +14,54 @@ export default (vueElement, googleMapsElement, props, options) => {
     const getMethodName = 'get' + capitalizeFirstLetter(attribute);
     const eventName = attribute.toLowerCase() + '_changed';
 
-    // We need to avoid an endless
-    // propChanged -> event emitted -> propChanged -> event emitted loop
-    // although this may really be the user's responsibility
-    var timesSet = 0;
-
-    vueElement.$watch(attribute, () => {
-      const attributeValue = vueElement[attribute];
-
-      timesSet++;
-      googleMapsElement[setMethodName](attributeValue);
-      if (afterModelChanged) {
-        afterModelChanged(attribute, attributeValue);
-      }
-    }, {
-      deep: type === Object
-    });
-
-    if (twoWay) {
-      googleMapsElement.addListener(eventName, (ev) => {
-          if (timesSet > 0) {
-            timesSet --;
-            return;
+    if (!twoWay) {
+      vueElement.$watch('local_'+attribute, () => {
```

Comment:
This isn't going to work is it? You're watching your generated values, but nothing is going to set it